pyfar/plot/plotstyles/dark.mplstyle

```diff
@@ -20,9 +20,9 @@ font.style          : normal
 font.variant        : normal
 font.weight         : normal
 font.stretch        : normal
-font.size           : 10.0
```

Comment:
I would not specify font sizes, they are resolution/display specific

Reply:
see above